src/types/eip.ts

```diff
@@ -0,0 +1,26 @@
+export type EIP = {
+  number: number;
+  title: string;
+  link: string;
+  status: EIPState; // The status should always be `Final` for now.
+  activeHardforks: string[];
+  deprecated?: boolean;
+  // Some EIPs have parameters, such as EIP-1559, but these parameters may not be the same on all
+  // chains. This field is intended to list the names and values of any parameters that exist.
+  parameters?: EIPParameter[];
```

Comment:
Let's add a `references` field to the `EIP` type that holds any other useful references, such as sources for the parameters, and have those included in the "References" drop-down in the UI

Reply:
Done in https://github.com/mds1/evm-diff/pull/50/commits/48e42de8ffcf49774575be2e9d2039ea5a6b2754.